content/shmem_sync_all.tex

```diff
@@ -0,0 +1,44 @@
+\color{ForestGreen}
+\apisummary{
+    Registers the arrival of a \ac{PE} at a barrier and suspends \ac{PE}
```

Comment:
I understand the difference :) Probably, the confusion is because of the usage of “suspend”.  

Suggestion : suspend -> block the calling thread